### ISSUE TITLE
TinyGo Wasm :)

### ISSUE THREAD

**Issue by justinclift** [cretz/go-wasm-bake#1]
As a potentially interesting data point, some of the people from the Go #WebAssembly slack channel have started doing stuff with TinyGo's (very) new WebAssembly generator:

&nbsp; &nbsp; https://github.com/aykevl/tinygo

TinyGo is (for now) a sub set of Go aimed at embedded devices... which turned out to have a lot in common with a WebAssembly execution environment. :wink:

The main thing though, is that it uses LLVM 7 (which does Wasm) for code generation.  And that automatically eliminates dead code.  So the resulting Wasm sizes (thus far, early days yet) are super tiny.  Less than 1k for toy examples.

**Comment by cretz** [comment]
Neat. IMO, Go is a poor language whose primary benefit is the standard library and libraries built on top of that. Were I writing WASM from scratch sans dependencies though, I would have a look at TinyGo et al.

**Comment by justinclift** [comment]
Yeah.  The TinyGo approach seems to be a good avenue for now. :smile:

I'll close this issue, as it's really just a FYI/interesting-pointer thing, and it doesn't need to be left open. :smile:
